### DISCUSSION TITLE
Analog sticks when?

### DISCUSSION THREAD

**Discussion by Oppa-YA** [OpenStickFoundation/GP2040-CE#160] [Ideas]
Hello, your project looks great but why it does not have analog stick support? Can you please add it?

**Comment by TheTrainGoes** [comment]
Hi @Oppa-YA ,

We do currently have analog stick support (currently just one).  It will take some tinkering to get it working perfectly based on your specific analog stick.  You can check it out more with the most recent compile in web-config.

**Reply by Oppa-YA** [reply]
Oh thank you, I will check it. Does I need latest release or some beta version? I tried some time ago but it didn't had it at that time.
